src/main/java/Parser.java

```diff
@@ -0,0 +1,234 @@
+
+import java.sql.SQLOutput;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+public class Parser {
+
+    private Ui user;
+    private ArrayList<Task> items;
+
+    /**
+     * Constructor method for Parser.
+     * @param user object handling all messages
+     */
+    public Parser(Ui user) {
+        this.user = user;
+        this.items = new ArrayList<>();
+    }
+
+    //return Ui obj
+    public Ui getUser() {
+        return this.user;
+    }
+
+    //return list of tasks
+    public ArrayList<Task> getList() {
+        return this.items;
+    }
+
+    /**
+     * Deleting of task from ArrayList.
+     * @param num index of task in list
+     * @return String delete message
+     * @throws IndexOutOfBoundsException index of task not found in list
+     *
```

Comment:
improper javadocs style
use `Deletes` instead of `Deleting`,
use consistent line breaks at the end of javadoc